model/godleyTableWindow.cc

```diff
@@ -347,41 +349,44 @@ namespace minsky
               highlightColumn(surface->cairo(),selectedCol);
               highlightColumn(surface->cairo(),motionCol);
             }
-          else if (motionRow>=0 && selectedCol==0 && selectedRow>0) // whole col being moved
+          else if (motionRow>=0 && selectedCol==0 && selectedRow>0) // whole Row being moved
             {
               highlightRow(surface->cairo(),selectedRow);
               highlightRow(surface->cairo(),motionRow);
             }
           else if (selectedCol==0 || /* selecting individual cell */
-                   (selectedCol>=int(scrollColStart) && selectedCol<int(godleyIcon->table.cols())))
+                   (selectedCol>=int(scrollColStart) && selectedCol<int(godleyIcon->table.cols())))   
             {
-              if (selectedRow!=0 || selectedCol!=0) // can't select flows/stockVars label
+              if (selectedRow!=0 || selectedCol!=0) // can't select flows/stockVars
                 {
```

Comment:
I think this would be more simply achieved by changing this condition to
if (selectedRow>1 || selectedCol!=0)
rather than nesting the conditional.

Need to check if selectedRow could be negative.

Reply:
Great, thanks, it works perfectly. I've made it 
 if ((selectedRow>1 || selectedRow <0) || selectedCol!=0)
to check whether selectedRow could also be negative.